### PR TITLE
Add missing values to PricingUnit enum

### PR DESCRIPTION
**Pull request by zhmaeff**
JIRA ticket: N/A

Added missing values for PricingUnit.

See [Supported Metered Usage Units](https://docs.appdirect.com/developer/apis/distribution-api-guide/metered-usage-api)

- [ ] Approved by a PI tech lead

### CODE REVIEW

File: src/main/java/com/appdirect/sdk/appmarket/events/PricingUnit.java [hunk 1]
```diff
@@ -20,13 +20,247 @@
...
 	PROVIDER,
 	MANAGER,
+	TESTER,
+	JVM,
+	SERVER,
+	WEB_USE_MINUTE,
+	AUDIO_USE_MINUTE,
+	PIECE,
+	EMPLOYEE_PAY_PERIOD,
+	EMPLOYEE_PER_PAY_PERIOD,
```
Comment (fedesassone) [inline src/main/java/com/appdirect/sdk/appmarket/events/PricingUnit.java]: should we add this value to doc center?

File: src/main/java/com/appdirect/sdk/appmarket/events/PricingUnit.java [hunk 2]
```diff
@@ -20,13 +20,247 @@
...
+	COMPUTER,
+	NOT_APPLICABLE,
```
Comment (fedesassone) [inline src/main/java/com/appdirect/sdk/appmarket/events/PricingUnit.java]: and this one?

File: src/main/java/com/appdirect/sdk/appmarket/events/PricingUnit.java [hunk 3]
```diff
@@ -20,13 +20,247 @@
...
+	ONE_TIME_SETUP,
```
Comment (fedesassone) [inline src/main/java/com/appdirect/sdk/appmarket/events/PricingUnit.java]: and this one?

File: src/main/java/com/appdirect/sdk/appmarket/events/PricingUnit.java [hunk 4]
```diff
@@ -20,13 +20,247 @@
...
+	SMS_TEXT_MESSAGE,
+	CONTACTS_1000,
+	CONTRACT_FEE,
+	TRANSFER_FEE,
+	REACTIVATION_FEE,
+	RECIPIENT,
+	ADDITIONAL_1000_CONTACTS_BLOCK,
+	SCHEDULE_PLAN,
+	EMAILS_1000,
+	EMAILS_2500,
```
Comment (fedesassone) [inline src/main/java/com/appdirect/sdk/appmarket/events/PricingUnit.java]: And this one?

File: src/main/java/com/appdirect/sdk/appmarket/events/PricingUnit.java [hunk 5]
```diff
@@ -20,13 +20,247 @@
 	USER,
 	GIGABYTE,
 	MEGABYTE,
-	PROPERTY,
```
Comment (fedesassone) [inline src/main/java/com/appdirect/sdk/appmarket/events/PricingUnit.java]: This one looks as supported on the doc

Comment (fedesassone) [reply src/main/java/com/appdirect/sdk/appmarket/events/PricingUnit.java]: meaning PROPERTY

File: src/main/java/com/appdirect/sdk/appmarket/events/PricingUnit.java [hunk 6]
```diff
@@ -20,13 +20,247 @@
...
+	REPORT,
+	BOX,
+	SESSION,
+	DISPLAY,
+	TRUCKROLL,
+	TRANSACTION_FEE,
+	SENDING_API_CALL,
+	LOOKUP_API_CALL,
+	ANALYTICS_API_CALL,
+	MIGRATION_INSTANCE,
```
Comment (fedesassone) [inline src/main/java/com/appdirect/sdk/appmarket/events/PricingUnit.java]: Also adding this one and all the ones below? Where did we retrieve the needed "missing values" from?

Comment (zhmaeff) [reply src/main/java/com/appdirect/sdk/appmarket/events/PricingUnit.java]: All missing values were taken from Docs (https://docs.appdirect.com/developer/apis/distribution-api-guide/metered-usage-api)

File: src/main/java/com/appdirect/sdk/appmarket/events/PricingUnit.java (hunk 1 above)
Comment (zhmaeff) [reply src/main/java/com/appdirect/sdk/appmarket/events/PricingUnit.java]: Please, take a look at this doc page (https://docs.appdirect.com/developer/apis/distribution-api-guide/metered-usage-api).
```PricingUnit``` enum values were taken from "Supported Metered Usage Units" list.

File: src/main/java/com/appdirect/sdk/appmarket/events/PricingUnit.java (hunk 5 above)
Comment (zhmaeff) [reply src/main/java/com/appdirect/sdk/appmarket/events/PricingUnit.java]: `PROPERTY` just moved to line 28.

**Comment by zhmaeff** [comment]
@davidchau Is there anything else I can do in this PR to have it merged?

**Pull request merged**